phc/offset_test.go

```diff
@@ -56,6 +56,49 @@ func TestSysoffEstimateExtended(t *testing.T) {
 	require.Equal(t, want, got)
 }
 
+func TestSysoffFromExtendedTS(t *testing.T) {
+	extendedTS := [3]PTPClockTime{
+		{Sec: 1667818190, NSec: 552297411},
+		{Sec: 1667818153, NSec: 552297462},
+		{Sec: 1667818190, NSec: 552297522},
+	}
+	sysoff := sysoffFromExtendedTS(extendedTS)
+	want := SysoffResult{
+		SysTime: time.Unix(1667818190, 552297466),
+		PHCTime: time.Unix(1667818153, 552297462),
+		Delay:   111,
+		Offset:  37000000004,
+	}
+	require.Equal(t, want, sysoff)
+}
+
+func TestOffsetBetweenExtendedReadings(t *testing.T) {
+	extendedA := &PTPSysOffsetExtended{
+		NSamples: 5,
```

Comment:
Could you make it 6 just to be sure we test the NSamples comparison logic?

Reply:
Done